figma-shop/src/app/app.module.ts

```diff
@@ -10,18 +10,20 @@ import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 
 import { HeaderModule } from './components/header/header.module';
+import { FilterModule } from './shared/pipes/filter/filter.module';
```

Comment:
This import produce a building error! There is no directory 'pipes' in this branch

Reply:
Done